### PR TITLE
Feature/homescreen

### PR DESCRIPTION
**Pull request by ViktorGrigorov2910**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,6 +12,19 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.MoviO"
-        tools:targetApi="31" />
+        tools:targetApi="31">
+        <activity
```
Comment (Golemanova) [inline app/src/main/AndroidManifest.xml:16]: What is this activity for?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">moviO</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (Golemanova) [inline app/src/main/res/values/strings.xml]: I don't see this used anywhere. Do you need it?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ViktorGrigorov2910) [reply app/src/main/AndroidManifest.xml:16]: It was for demo/testing and I forgot to delete it.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (ViktorGrigorov2910) [reply app/src/main/res/values/strings.xml]: I don't need it. Also forgot to delete

**Review by Golemanova — APPROVED** [review]

**Pull request merged**